encoder.go

```diff
@@ -98,7 +98,7 @@ func (kse *keyStoreEncoder) writeCertificate(cert *Certificate) error {
 		return err
 	}
 	certLen := len(cert.Content)
```

Comment:
I think wrapping `uint64(len(cert.Content))` would be enough.
Could you fix it and I'll merge it then.